.github/workflows/publish_nightly.yml

```diff
@@ -0,0 +1,52 @@
+name: Publish Nightly
+on:
+  schedule:
+    - cron: '0 6 * * *' # Daily at 6AM UTC, 10PM PT
```

Comment:
Following the README instructions, people will be able to use Nightly builds from tomorrow onwards because the first nightly build will be generated tomorrow (at 0600 UTC). Right?

Reply:
In theory yes, but I'll try to trigger one automatically during the day today after we merge this